glide.yaml

```diff
@@ -187,7 +186,7 @@ import:
 
 # required by opencontainers
 - package: github.com/sirupsen/logrus
-  version: 839c75faf7f98a33d445d181f3018b5c3409a45e
+  version: v1.4.2
```

Comment:
okay - it is on the same level

Reply:
Yes, just updating to what kube was using.